modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,131 @@
+
+* In the Red Hat OpenShift Pipelines 1.9 release, the `tkn pipelinerun describe` command does not display timeouts for the `PipelineRun` resource.
+
+* In the Red Hat OpenShift Pipelines 1.9 release, the the `PipelineResource` custom resource (CR) is deprecated. The `PipelineResource` CR was a Tech Preview feature and part of the `tekton.dev/v1alpha1` API.
+
+* In the Red Hat OpenShift Pipelines 1.9 release, custom image parameters in cluster tasks are deprecated. You must run an operator-provided cluster task with a custom image. The following parameters must be configured in the custom image:
+
++
+[source,yaml]
+----
+buildah task doesn't expose "BUILDER_IMAGE" param
+git-clone task doesn't expose "gitInitImage" param
+kn task doesn't expose "kn-image" param
+kn-apply task doesn't expose "KN_IMAGE" param
+maven task doesn't expose "MAVEN_IMAGE" param
+s2i-* tasks don't expose "BUILDER_IMAGE" param
+tkn task doesn't expose "TKN_IMAGE" param
+----
+
+[id="known-issues-1-9_{context}"]
+== Known issues
+
+// .Operator
+
+* The `chains-secret` and `chains-config` config map are removed after you uninstall the Red Hat OpenShift Pipelines operator. As they contain user data, they should be preserved and not deleted.
+
+// https://issues.redhat.com/browse/SRVKP-2396
+
+[id="fixed-issues-1-9_{context}"]
+== Fixed issues
+
+// .Pipelines
+
+* With this update, after replacing the empty array, the original array will be empty.
+// Vincent Demeester
+
+* Before this update, if duplicate secrets are present in a service account for a pipelines run, it resulted in failure in task pod creation. With this update, this issue is resolved and the task pod is created successfully even if duplicate secrets were present in a service account.
+// Vincent Demeester
+
+* Before this update, by looking at the TaskRun's `spec.StatusMessage` field, users could not distinguish whether the `TaskRun` had been cancelled by the user or by a `PipelineRun` that was part of it. With this update, this issue is resolved and users can distinguish the status of the `TaskRun` by looking at the TaskRun's `spec.StatusMessage` field.
+// Vincent Demeester
+
+* With this update, the user can specify tasks or timeout.
+// Vincent Demeester
+
+* Before this update, a user could not remove webhook validation when deleting old versions of invalid objects. With this update, this issue is resolved and user can remove webhook validation when deleting old versions of invalid objects.
+// Vincent Demeester @vdeemester
+
+* With this update, the user can merge labels and annotations instead of override.
+// Vincent Demeester @vdeemester
+
+// .Triggers
+
+* Before this update, log keys did not have same keys as in pipelines controllers. With this update, this issue has been resolved and the log keys have been updated to match the log stream. The keys in logs have been changed from "ts" to "timestamp", from "level" to "severity", and "message" to "msg".
```

Comment:
``log keys have been updated to match the log stream`` 
Should it be to match the log stream of pipelines controllers?

Reply:
If you say it should be pipelines controllers, I can make a change. The text provided in the SME doc is as follows:

Update log keys to match logstream:  Log lines formatted as JSON have the severity in "severity" (was "level"), timestamp in "timestamp" (was "ts"), and message in "message" (was "msg").

Nowhere is it mentioned to match the log stream of pipeline controllers. Please suggest.

Reply:
Yes, it was to match the log stream of pipeline controllers. Triggers and Pipelines have different keys, this will make it same for both.

Reply:
Thanks for the clarification, @khrm. I have made the change as per your suggestion. Kindly confirm.